java/src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -184,7 +192,8 @@ public static ValidateAIF createForDWD(Restriction restriction) {
         restrictions.add("com/ncc/aif/dwd_aif.shacl");
         switch (restriction) {
             case NIST_TA3:
-                restrictions.add(NIST_HYPOTHESIS_SHACL_RESNAME);
+                restrictions.add(NIST_CLAIMFRAME_SHACL_RESNAME);
+                //restrictions.add(NIST_HYPOTHESIS_SHACL_RESNAME);
```

Comment:
Should we comment out or erase references to hypothesis?

Reply:
removed